### ISSUE TITLE
Show a DotPlot for when the duplication validation fails

### ISSUE THREAD

**Issue by IsmailM** [wurmlab/genevalidator#111]
When the wilcoxon test suggest that a sequence has been duplicated, it may be useful to show users a Dotplot (i.e. a sequence against itself). This would inform users in a highly visual manner, the region of the sequence that is duplicated and the rough percentage of the sequence that has been duplicated.

In order to avoid noise in DotPlot, one could simply use a window size equivalent to the duplicated region.

**Comment by yannickwurm** [comment]
agreed (but could be shown in all situations) - much like our other graphs. 
maybe the biojs folks have a good dotplot library?

**Comment by yeban** [comment]
+1.

I did a quick search through BioJS registry and didn't find anything relevant. @IsmailM what would it take to do dot-plots in the browser? I'm interested in this for SequenceServer as well.

**Comment by yannickwurm** [comment]
(I'm sure there are other webby implementations (I remember seeing one from a german group 2-3 years ago... but forget the name) - we should not reinvent the wheel here)
